syntax = "proto3";

package adlik.serving;
option cc_enable_arenas = true;

message KMeansConfig {}

message AmcConfig {
  double nBler_target = 1;
  double lambda = 2;
}

message AlgorithmConfig {
  oneof config {
    KMeansConfig kmeans = 1;
    AmcConfig amc_config = 2;
  }
}
